meshstack-hub/.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
```

Comment:
d: Please remove ts specific configs. This will be covered via eslint.

Reply:
Can you please remove the whole [*.ts] section ?

Reply:
done